.circleci/config.yml

```diff
@@ -27,6 +30,9 @@ base_job: &base_job
 test_job: &test_job
   steps:
     - checkout
+    - run:
+        name: SPM SSH Workaround
+        command: *spm_ssh_workaround
```

Comment:
what does the asterisk in front of the variable suggest here?

Reply:
It references the definition at the top, just yaml's way of making things a little more reusable.